frontend/public/components/deployment-config.tsx

```diff
@@ -47,52 +47,56 @@ const menuActions = [
   ...common,
 ];
 
-export const DeploymentConfigsDetails: React.SFC<{obj: any}> = ({obj: deploymentConfig}) => {
+export const DeploymentConfigOverviewList = ({deploymentConfig}) => {
```

Comment:
Let's pull this out of the PR for the moment and work on the details sidebar in a separate PR